packages/bot-web-ui/src/components/download/download.tsx

```diff
@@ -27,15 +27,14 @@ const Download = observer(({ tab }: TDownloadProps) => {
             classNameBubble='run-panel__info--bubble'
             alignment='bottom'
             message={popover_message}
-            zIndex={5}
+            zIndex='5'
         >
             <Button
                 id='download-button'
                 is_disabled={is_clear_stat_disabled}
-                className='download__button'
-                icon={
-                    <Icon icon='IcDownload' color={is_clear_stat_disabled && 'disabled'} className='download__icon' />
```

Comment:
we dont need the disabled icon anymore?

Reply:
It's actually handled by the button itself (is_disabled)